README.md

```diff
@@ -212,6 +212,9 @@ Find the ACORD year/month at the bottom left corner of each ACORD page. Often th
 
 - [CMS 1500 (08/05)](./health_insurance_claims/cms_1500/cms_1500_08_05.json)
 
+### NLP
```

Comment:
Let's call this "HR" instead, we should categorize these by use case rather than the methods that we used to set it up.